src/libopenrave/environment.cpp

```diff
@@ -120,8 +121,8 @@ void EnvironmentBase::EnvironmentBaseInfo::DeserializeJSONWithMapping(const rapi
         orjson::LoadJsonValueByKey(rEnvInfo, "revision", _revision);
     }
 
-    if (rEnvInfo.HasMember("unit")) {
-        orjson::LoadJsonValueByKey(rEnvInfo, "unit", _unit);
+    if (rEnvInfo.HasMember("unitInfo")) {
```

Comment:
You have to continue to support the old "unit" since there are too many environments out there that use it.

Reply:
sure. I add it back